test/unit/playerutils-test.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'utils/playerutils',
+    'version'
+], function (playerutils, version) {
+    /* jshint qunit: true */
+
+    module('playerutils');
+
+    test('playerutils.versionCheck', function(assert) {
+        var versionCheck = playerutils.versionCheck('0 5');
+        assert.ok(versionCheck, 'playerutils.versionCheck with valid version');
+
+        versionCheck = playerutils.versionCheck('10 1');
+        assert.notOk(versionCheck, 'playerutils.versionCheck with invalid version');
+
+        var jParts = version.split(/\W/);
```

Comment:
@donato This does not work because playerutils is already initialized and use current version number.